linux/clipboard_watcher_plugin.cc

```diff
@@ -25,26 +20,25 @@ G_DEFINE_TYPE(ClipboardWatcherPlugin,
 void handle_owner_change(GtkClipboard* clipboard,
                          GdkEvent* event,
                          gpointer data) {
-  if (plugin_instance->is_watching != true) {
+  ClipboardWatcherPlugin* plugin = CLIPBOARD_WATCHER_PLUGIN(data);
+  if (plugin->is_watching != true) {
     return;
   }
 
   g_autoptr(FlValue) result_data = fl_value_new_map();
-  fl_method_channel_invoke_method(plugin_instance->channel,
+  fl_method_channel_invoke_method(plugin->channel,
                                   "onClipboardChanged", result_data, nullptr,
                                   nullptr, nullptr);
 }
 
 static FlMethodResponse* start(ClipboardWatcherPlugin* self, FlValue* args) {
   self->is_watching = true;
-  return FL_METHOD_RESPONSE(
-      fl_method_success_response_new(fl_value_new_bool(true)));
+  return FL_METHOD_RESPONSE(fl_method_success_response_new(nullptr));
```

Comment:
alternatively, we could use `g_autoptr()` to ensure that the result value is released
```c
g_autoptr(FlValue) result = fl_value_new_bool(true);
return FL_METHOD_RESPONSE(fl_method_success_response_new(result));
```
however, the result is not being handled so therefore, i proposed to simply remove the unnecessary allocation:
https://github.com/leanflutter/clipboard_watcher/blob/6c3d9392358d8866de5196c1512dcb7b270b7e8a/lib/src/clipboard_watcher.dart#L53-L59